crates/autopilot/src/run_loop.rs

```diff
@@ -175,14 +172,11 @@ impl RunLoop {
                 return;
             }
 
-            tracing::info!("executing with solver {}", index);
-            match self
-                .execute(auction, id, &self.drivers[index], &solution)
-                .await
-            {
+            tracing::info!("settling with solver {}", index);
```

Comment:
Not related to this PR but I noticed that only logging the index of the solver is pretty bad for debugging.
Would be nice to have the name here.
Also this and the next log should have the `index` as an tracing argument instead of being baked into the string.

Reply:
Agreed, this is pretty bad.